sklearn/utils/fixes.py

```diff
@@ -21,6 +21,13 @@
 from .._config import config_context, get_config
 from ..externals._packaging.version import parse as parse_version
 
+# below copied verbatim from scipy._lib._util.py to be used in rng_integers
+try:
+    from numpy.random import Generator as Generator
```

Comment:
Our minimum supported NumPy version is 1.17.3, so we can assume that Generators can be imported.